client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -168,7 +168,7 @@ class EditContactInfoFormCard extends React.Component {
 				</FormLabel>
 				<DesignatedAgentNotice
 					domainRegistrationAgreementUrl={ domainRegistrationAgreementUrl }
-					saveButtonLabel={ translate( 'Save Contact Info' ) }
+					saveButtonLabel={ translate( 'Save contact info' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.